### ISSUE TITLE
Building error only on El Capitan?

### ISSUE THREAD

**Issue by Tomon613** [ElonaFoobar/ElonaFoobar#163]
In recent codes, a few build error happens in cat.hpp, elona.hpp, calc.cpp, and putit.hpp with logs like the following.

```
#built in directory named “ElonaF”  

In file included from /src/ElonaF0421/ability.cpp:1:
In file included from /src/ElonaF0421/ability.hpp:7:
/src/ElonaF/cat.hpp:105:15: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
    void push(nullptr_t)
              ^~~~~~~~~
              std::nullptr_t
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/cstddef:97:31: note: 
      'std::nullptr_t' declared here
    typedef decltype(nullptr) nullptr_t;
                              ^
```

Listed some “nullptr_t” fixed to “std::nullptr_t”, the make command goes well.

Anyone knows what causes these errors? 

(I think these are maybe caused by Xcode8.2, the latest version on El Capitan, which cannot treat Xcode8.3, installed in Sierra.)   

# Environment

* OS version: OSX El Capitan (MacBookPro early 2015)
* Elona Foobar verision: latest (2018.4.22)
* Xcode: 8.2.1

**Comment by ki-foobar** [comment]
This error may be compiler dependant. I think that my compiler defines `nullptr_t` through included headers but your compiler does not. I'll fix `nullpt_t` to `std::nullptr_t`. Thank you for contributing.

**Comment by ki-foobar** [comment]
@Tomon613 
I fixed it at #166. However, my compiler was able to build Foobar before, so could you checkout branch and tell me whether you are able to build this one successfully?

```sh
$ git clone https://github.com/ki-foobar/ElonaFoobar
$ cd ElonaFoobar
$ git checkout feature/add-std-namespace-to-nullptr_t
$ make
success or failure?
```

**Comment by Tomon613** [comment]
@ki-foobar 
The make command went on your fixed source smoothly!
But launching failed like #141.
Thanks.
